codegenerator/cli/src/config_parsing/chain_helpers.rs

```diff
@@ -260,80 +262,26 @@ impl Network {
     //TODO: research a sufficient threshold for all chain (some should be 0)
     pub fn get_confirmed_block_threshold(&self) -> i32 {
         match self {
-            Network::EthereumMainnet
-            | Network::Goerli
-            | Network::Optimism
-            | Network::Base
-            | Network::BaseSepolia
-            | Network::Bsc
-            | Network::PoaSokol
-            | Network::Chapel
-            | Network::PoaCore
-            | Network::Gnosis
-            | Network::Fuse
-            | Network::Fantom
-            | Network::Polygon
-            | Network::Boba
-            | Network::OptimismGoerli
+            Network::OptimismGoerli
             | Network::OptimismSepolia
-            | Network::Clover
-            | Network::Moonbeam
-            | Network::Moonriver
-            | Network::Mbase
-            | Network::FantomTestnet
+            | Network::Optimism
             | Network::ArbitrumOne
             | Network::ArbitrumNova
             | Network::ArbitrumGoerli
-            | Network::ArbitrumSepolia
-            | Network::Celo
-            | Network::Fuji
-            | Network::Avalanche
-            | Network::CeloAlfajores
-            | Network::Mumbai
-            | Network::Aurora
-            | Network::AuroraTestnet
-            | Network::Harmony
-            | Network::BaseGoerli
-            | Network::ZksyncEra
-            | Network::Sepolia
-            | Network::Linea
-            | Network::Rinkeby
-            | Network::ZksyncEraTestnet
-            | Network::PolygonZkevmTestnet
-            | Network::PolygonZkevm
-            | Network::ScrollSepolia
-            | Network::Scroll
-            | Network::Metis
-            | Network::Manta
-            | Network::TaikoJolnr
-            | Network::Kroma
-            | Network::Lukso
-            | Network::XLayerTestnet
-            | Network::XLayer
-            | Network::Holesky
-            | Network::GnosisChiado
-            | Network::Zora
-            | Network::PublicGoods
-            | Network::A1Milkomeda
-            | Network::C1Milkomeda
-            | Network::Flare
-            | Network::Mantle
-            | Network::Zeta
-            | Network::BerachainArtio
-            | Network::NeonEvm
-            | Network::Rsk
-            | Network::ShimmerEvm
-            | Network::Blast
-            | Network::BlastSepolia
-            | Network::FhenixTestnet
-            | Network::Amoy
-            | Network::Crab
-            | Network::Darwinia
-            | Network::Cyber => DEFAULT_CONFIRMED_BLOCK_THRESHOLD,
+            | Network::ArbitrumSepolia => 0,
+
+            _ => DEFAULT_CONFIRMED_BLOCK_THRESHOLD,
```

Comment:
I was trying to avoid doing this so that if new chains get added one has to consciously set the confirmed block threshold. Would prefer to remove the `_` default.